doc-content/enterprise-only/admin-and-config/managing-business-central-configuring-githooks-proc.adoc

```diff
@@ -0,0 +1,100 @@
+[id='managing-business-central-configuring-githooks-proc']
+= Configuring git hooks
+
+{CENTRAL} can automatically push changes to a remote repository using Git hooks. After you configure the post-commit hook, {CENTRAL} automatically pushes your changes to the remote repository.
+
+[NOTE]
+====
+Currently only post-commit hooks are supported. Post-commit hooks are triggered after the commit process finishes.
```

Comment:
maybe a link to office Git hook docs like here https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks

Reply:
Added reference as suggested.